runtime/worker.rs

```diff
@@ -208,6 +209,16 @@ impl MainWorker {
       },
     );
 
+    let op_summary_metrics = Rc::new(OpMetricsSummaryTracker::default());
+    deno_core::extension!(deno_op_metrics,
+      options = {
+        op_summary_metrics: Rc<OpMetricsSummaryTracker>
+      },
+      state = |state, options| {
+        state.put(options.op_summary_metrics)
+      }
+    );
```

Comment:
Is it only for `MainWorker`? What about `WebWorker`?

Also seems strange to define an extension to put a default object in the state. Maybe do it directly after creating ` JsRuntime`?

Reply:
I added code for WebWorker as well -- the metrics there are never fetched (AFAICT) but I suppose it's good for consistency. 

I switched it up to manually add it to the JsRuntime after as suggested but `deno_permissions_worker` is also doing something similar. I'm OK either way.